.github/workflows/build_matrix.yml

```diff
@@ -17,6 +17,20 @@ on:
         required: false
         default: true
         type: boolean
+  workflow_dispatch:
+    inputs:
+      ref:
```

Comment:
This is an oversight, but can you provide documentation here within the YML to explain what the `ref` and especially `id` parameters are expected to be?

The ref is probably mostly self-explanatory (something like "main" or a commit hash should be acceptable?) but `id` is cryptic.

Reply:
Done